File: URI/Beginner/Python/1008_Salary.py
#=====================================    
#   URI Online Judge Problem
#   Category: Begginer
#   Problem: 1008 - Salary
#   Status: Accepted
#=====================================

number = int(input())
hours = int(input())
amountPerHour = float(input())

print("NUMBER =", number)
print("SALARY = U$ {:.2f}".format(hours*amountPerHour))

File: URI/Beginner/C/1008_Salary.c
/*
	URI Online Judge Problem
	Category: Begginer
	Problem: 1008 - Salary
	Status: Accepted
*/

#include <stdio.h>

int main() {
	int number, hours;
	float amtPerHour;
	
	scanf("%d", &number);
	scanf("%d", &hours);
	scanf("%f", &amtPerHour);
	
	printf("NUMBER = %d\n", number);
	printf("SALARY = U$ %.2f\n", hours*amtPerHour);
	
	return 0;
}
